utils/src/main/java/com/cloud/utils/StringUtils.java

```diff
@@ -313,4 +313,42 @@ public static String mapToString(final Map<String, String> map) {
     public static String toCSVList(final List<String> csvList) {
         return join(csvList, ",");
     }
+
+    public static int min(int x, int y, int z)
+    {
+        if (x <= y && x <= z)
+            return x;
+        if (y <= x && y <= z)
```

Comment:
```suggestion
        if (y <= z)
```

Reply:
I've taken out minimum method and used min(min(x,y),z)